src/stepper.js

```diff
@@ -1,6 +1,12 @@
+/* @flow */
 const errorMargin = 0.0001;
 
-export default function stepper(frameRate, x, v, destX, k, b) {
+export default function stepper(frameRate: number,
+                                x: number,
```

Comment:
Sorry for nitpicking but I can't recall us using this awkward indentation and it will not pass ESLint check.

Should be something like 

```js
export default function stepper(
  frameRate: number,
  x: number,
  v: number,
  destX: number,
  k: number,
  b: number): [number, number] {
```

Disregard if I'm wrong.